.github/workflows/ci.yaml

```diff
@@ -33,27 +35,33 @@ jobs:
         uses: shivammathur/setup-php@v2
         with:
           php-version: ${{ matrix.php-version }}
-          coverage: pcov
+          coverage: ${{ matrix.coverage-driver }}
           tools: composer:v2
 
+      - name: Get composer cache directory
+        id: composer-cache
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
       - name: Cache dependencies
         uses: actions/cache@v2
         with:
-          path: ~/.cache/composer
-          key: composer-${{ matrix.php-version }}-${{ hashFiles('**/composer.*') }}-${{ matrix.dependencies }}
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: composer-${{ runner.os }}-${{ matrix.php-version }}-${{ hashFiles('**/composer.*') }}-${{ matrix.dependencies }}
           restore-keys: |
-            composer-${{ matrix.php-version }}-${{ hashFiles('**/composer.*') }}-
-            composer-${{ matrix.php-version }}-
+            composer-${{ runner.os }}-${{ matrix.php-version }}-${{ hashFiles('**/composer.*') }}-
+            composer-${{ runner.os }}-${{ matrix.php-version }}-
+            composer-${{ runner.os }}-
             composer-
 
       - name: Install dependencies
         run: |
           composer update --no-interaction --prefer-dist --no-progress ${{ matrix.dependencies }}
 
-      - name: Run tests
+      - name: Run tests using Makefile
+        if: runner.os != 'Windows'
         run: |
           make test-unit
```

Comment:
Running PHPUnit on Windows is super annoying. It's either a .BAT, or some other BS I don't think it's worth dealing right now. More importantly Infection itself knows how to work around this, therefore this step is actually redundant. But let's keep it for now, not when under Windows.

Reply:
At the end of all the changes, does it mean we won't run unit tests on Windows? If so - why? currently we run them on AppVeyor and I would say we should continue doing that.

Reply:
Not at all! On the contrary, Infection will run them as it does. The initial test run, remember?

Reply:
Here: https://github.com/infection/infection/runs/1290189378#step:8:15

Reply:
The only real downside here is that we won't see skipped-incomplete tests in the log for Windows builds. Think it's a blocker?

Reply:
Right, we're running tests on Windows too now.

Reply:
sorry for the delay, good point with the initial tests run. But since you made an update to run tests on windows, let it be so ;)